go.mod

```diff
@@ -12,6 +12,8 @@ replace github.com/filecoin-project/test-vectors => ./extern/test-vectors // pro
 
 replace github.com/filecoin-project/filecoin-ffi => ./extern/filecoin-ffi // provided via a git submodule
 
+replace github.com/filecoin-project/go-state-types => github.com/filecoin-project/go-state-types v0.16.0-rc1 // provided via a git submodule
+
```

Comment:
`git rebase master` or merge master into this branch and you should get -rc1 for free